spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/ConfigCheck.scala

```diff
@@ -40,7 +47,8 @@ object ConfigCheck {
     }
   }
 
-  def getUnknownProperties(conf: SparkConf): Seq[String] = {
+  def getUnknownProperties(conf: SparkConf, extraProps: Set[String] = Set.empty): Seq[String] = {
```

Comment:
BTW: In defense of Javaism, I can say that having "get" in the beginning makes it somehow handy when naming things on the call site. Compare those two snippets:
```scala
def getUnknownProperties(...) = ???
// later in the code:
val unknownProperties = getUnknownProperties(...)  // obvious
```
```scala
def unknownProperties(...) = ???
val unknownProperties = unknownProperties(...)  // damn; now it requires either explicit type annotation, or a different name
```